tests/config/meilisearch.yaml

```diff
@@ -36,3 +36,7 @@ meilisearch:
             enable_serializer_groups: true
         -   name: self_normalizable
             class: 'Meilisearch\Bundle\Tests\Entity\SelfNormalizable'
+        -   name: dummy_custom_groups
+            class: 'Meilisearch\Bundle\Tests\Entity\DummyCustomGroups'
+            enable_serializer_groups: true
+            serializer_groups: ['public', 'private']
```

Comment:
I tried to change to `serializer_groups: ['public']` but the tests did not break, I was expecting to have a breaking in the last part:

Reply:
Did you cleared the cache? Because after removing `private` group I got:
```
1) Meilisearch\Bundle\Tests\Integration\CommandsTest::testImportsDummyWithCustomGroups
Failed asserting that two arrays are identical.
--- Expected
+++ Actual
@@ @@
         'objectID' => 1
         'id' => 1
         'name' => 'Dummy 1'
-        'createdAt' => '2024-04-04T07:32:01+00:00'
     )
     1 => Array &2 (
         'objectID' => 2
         'id' => 2
         'name' => 'Dummy 2'
-        'createdAt' => '2024-04-04T07:32:02+00:00'
     )
 )
```

Reply:
I think bundle should take care of clearing the cache before running locally, because I always need to clear it manually so changes are taken into account

Reply:
I think that was the issue, I tried your new version with the auto caching clear and it is working!